project_update_visible/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Project Update Visible",
+    "summary": """
+        Visible project_id in project.update view form.
+    """,
+    "author": "Odoo Community Association (OCA)",
```

Comment:
Add Binhex as author:

```suggestion
    "author": "Binhex, Odoo Community Association (OCA)",
```